modules/exploits/linux/http/trueonline_p660hn_v2_rce.rb

```diff
@@ -98,8 +98,8 @@ def exploit
     @cookie = rand_text_alpha_lower(7)
 
     res = send_request_cgi({
+      'method' => "GET",
```

Comment:
I think I made a mistake. I'll make changes as required. According to this https://github.com/rapid7/metasploit-framework/wiki/How-to-Send-an-HTTP-Request-Using-HTTPClient , we can add both get and post in send_request_cgi . So can I put both get and post in this case?